packages/integrations/vercel/README.md

```diff
@@ -72,6 +72,43 @@ import vercel from '@astrojs/vercel/serverless';
 import vercel from '@astrojs/vercel/static';
 ```
 
+### Middleware
```

Comment:
I’d suggest a slightly more descriptive title here. This will show up in search results on our docs site and would be a slightly misleading title in that context if someone was looking for more generic middleware information.

```suggestion
### Vercel Middleware
```